.buildkite/steps/publish.sh

```diff
@@ -134,7 +134,7 @@ generate_mappings() {
   local s3_mappings_cache
 
   image_id=$(buildkite-agent meta-data get image_id)
-  s3_mappings_cache="s3://${BUILDKITE_AWS_STACK_BUCKET}/mappings-${image_id}-${BUILDKITE_BRANCH}.yml"
+  s3_mappings_cache="s3://${BUILDKITE_STACK_BUCKET}/mappings-${image_id}-${BUILDKITE_BRANCH}.yml"
```

Comment:
Any reason for changing this variable name?

Reply:
Ahhh that's right, there are two different buckets

Reply:
`$BUILDKITE_AWS_STACK_BUCKET` was used here, and `$BUILDKITE_STACK_BUCKET` in the Makefile.
I only used `$BUILDKITE_STACK_BUCKET` because it was shorter.